README.rst

```diff
@@ -10,11 +10,17 @@ Google Cloud Platform (GCP) plugins for `gordon`_, an open-source, event-driven
 
 The ``gordon-gcp`` plugins add optional support for the following:
 
-* Creating, updating, and deleting records within `Google Cloud DNS`_
+Service:
+
 * Consuming events from `Google Cloud Pub/Sub`_
 * Reading from `Google Compute Engine`_ for record information
+* Creating, updating, and deleting records within `Google Cloud DNS`_
+
+Janitor:
 
-TODO (zephyr): what do the janitor plugins do?
```

Comment:
Sorry I missed this before. But in the future for "todos" in prose documentation, we use the [todo extention](http://www.sphinx-doc.org/en/master/ext/todo.html) ([example of our usage](https://raw.githubusercontent.com/spotify/gordon/develop/docs/plugins.rst)).